app/Account/AccountConstants.js

```diff
@@ -38,4 +38,4 @@ export const GET_USER_SUBSCRIPTION_DATA_FAIL = 'GET_USER_SUBSCRIPTION_DATA_FAIL'
 export const GET_USER_SUBSCRIPTION_DATA_SUCCESS = 'GET_USER_SUBSCRIPTION_DATA_SUCCESS';
 
 // Opt into/out of updates and promotions sendgrid emails
-export const ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE = 'ACCOUNT_DATA_EMAIL_PREFERENCES_CHECKBOX_CHANGE';
```

Comment:
Same deal, just double checking this isn't being used anywhere else anymore. If not, and the user can no longer change their email preferences in the extension panel or either of the hubs, we should be able to dispense with the `SUBSCRIBE_TO_EMAIL_LIST` action also, no?

Reply:
We actually don't have the functionality in the original hub or the panel to toggle sendgrid emails on, so this is only used here